### ISSUE TITLE
should create a deb not using zst compression

### ISSUE THREAD

**Issue by an0mal1a** [lsd-rs/lsd#891]
- os: Parrot OS 5.3
- `lsd --version`: 1.0.0

When try to install the .deb lsd:

dpkg-deb --info lsd_1.0.0_amd64.deb
dpkg-deb: error: el archivo `lsd_1.0.0_amd64.deb' contiene un miembro de datos `control.tar.zst' ininteligible, abandono

**Comment by marceldegraaf** [comment]
I have the same issue here, on Debian 11.7:

```
$ sudo dpkg -i lsd_1.0.0_amd64.deb

dpkg-deb: error: archive 'lsd_1.0.0_amd64.deb' uses unknown compression for member 'control.tar.zst', giving up
dpkg: error processing archive lsd_1.0.0_amd64.deb (--install):
 dpkg-deb --control subprocess returned error exit status 2
Errors were encountered while processing:
 lsd_1.0.0_amd64.deb
```

Seems related to Apt in Debian < 12 not supporting zstd: https://unix.stackexchange.com/questions/669004/zst-compression-not-supported-by-apt-dpkg

**Comment by rckgomz** [comment]
Same here.

**Comment by zwpaper** [comment]
Hi @marceldegraaf, thanks for the info, I searched from StackOverflow, and might found a solution, you could try to re-package it locally:

```bash
ar x lsd_1.0.0_amd64.deb # this is the only place to specify the lsd deb name
# Uncompress zstd files an re-compress them using xz
zstd -d < control.tar.zst | xz > control.tar.xz
zstd -d < data.tar.zst | xz > data.tar.xz
# Re-create the Debian package in /tmp/
ar -m -c -a sdsd ./lsd_1.0.0_local.deb debian-binary control.tar.xz data.tar.xz
# Clean up
rm debian-binary control.tar.xz data.tar.xz control.tar.zst data.tar.zst
```

and then install the lsd_1.0.0_local.deb

cc @rckgomz @an0mal1a

**Comment by zwpaper** [comment]
we use the latest ubuntu to do the package, it should be why zst is used to package deb.

**Comment by zwpaper** [comment]
fixed, will release in v1.1.0
